### ISSUE TITLE
Low image quality on ipad

### ISSUE THREAD

**Issue by vikueen12** [pookjw/SidecarPatcher#31]
I’ve managed to install sidecar but the quality of the image is pretty low and flickers. There’s no change if I plug in the lightening cable. Anyone has managed to get a high definition after installing the patch?

**Comment by ParthKetanDoshi** [comment]
Not a very high resolution but I changed the settings in System Preferences>Displays from Scaled to the only option available for manual selection (I guess it was 640x360 in my case). What I realised is when you mirror display, the quality seems to be better than using as a separate display.

**Comment by smk169** [comment]
I have similar issues of poor resulution. 
Selected scaled 1048X770  - no luck.
This happens both in wired and wireless.
Mac - early 2015 and ipad air (new 2019)

**Comment by ahmedifhaam** [comment]
Me too
Having similar issues with MacBook Pro 11,1 and iPad Pro 2018

**Comment by jienzang** [comment]
Yes, same issue MacBook Pro (Retina, 13-inch, early 2013) and iPad Pro 2018

**Comment by raghavthakur** [comment]
Same issue, poor image quality and screen flicker on iPad. Late-2013 MacBook Pro and iPad Pro 12.9" 2018

**Comment by ParthKetanDoshi** [comment]
Someone on YouTube has used **SwitchResX** Software to resolve this problem although am not very sure.
